pkg/platform/kube/platform.go

```diff
@@ -880,7 +888,7 @@ func (p *Platform) GetExternalIPAddresses() ([]string, error) {
 
 // ResolveDefaultNamespace returns the proper default resource namespace, given the current default namespace
 func (p *Platform) ResolveDefaultNamespace(defaultNamespace string) string {
-	if defaultNamespace == "@nuclio.selfNamespace" {
+	if defaultNamespace == "@nuclio.selfNamespace" || p.DefaultNamespace == "@nuclio.selfNamespace" {
```

Comment:
what's going on here?
why is there both `defaultNamespace` and `p.DefaultNamespace` ?
Looks like they need to be resolved to one variable with some precedence before looking at the contents